part-two/Population.md

```diff
@@ -1,24 +1,22 @@
 # Population
 
-Malthus was right about one thing. Global population did explode. And population growth started to accelerate right at the time of his writing around 1800.  
+Thomas Malthus is infamous for his theory of population growth and what has become known as a Malthusian catastrophe, or widespread famine as a result of the human population growing faster than agricultural production can keep up with {{"WIKIMS" | cite}}. He realized that a larger population means more scarcity, making it harder to meet everyone's basic needs. 
```

Comment:
I know you want to let people know about Malthus, but your sentence as currently written is awkward. For the start of a paragraph I like simple and direct sentences. Something like

In 18xx Thomas Malthus predicted widespread crises of famine and starvation as population growth outstrips humanity's ability to grow food. Malthus prediction was half right: Global population